support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -100,6 +100,45 @@ export const tests: Tests = {
 		referrerControlled: false,
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		seed: 3,
+		omitCountries: [
```

Comment:
Hey, is it alright to call this omitCountries, or should we be more specific as to why we are omitting these countries, incase there is a need to omit a different list of countries for another reason.